packages/calcite-components/src/components/tab/tab.scss

```diff
@@ -23,22 +24,33 @@
 
 .content {
   @apply box-border;
-  padding-block: var(--calcite-internal-tab-content-block-padding);
 }
 
 .scale-s {
-  --calcite-internal-tab-content-block-padding: var(--calcite-tab-content-block-padding, theme("spacing.1"));
-  @apply text-n2h;
+  font-size: var(--calcite-font-size-sm);
+  line-height: 1rem; //TODO: replace with token
+
+  .content {
+    padding-block: var(--calcite-tab-content-space, var(--calcite-tab-content-block-padding, theme("spacing.1")));
+  }
 }
 
 .scale-m {
-  --calcite-internal-tab-content-block-padding: var(--calcite-tab-content-block-padding, theme("spacing.2"));
-  @apply text-n1h;
+  font-size: var(--calcite-font-size);
+  line-height: 1rem; //TODO: replace with token
```

Comment:
I think we can remove the "TODO"s. I've created an issue https://github.com/Esri/calcite-design-system/issues/8856